lib/reader/src/parser.rs

```diff
@@ -158,6 +162,14 @@ impl<'a> Context<'a> {
         self.map.def_ebb(src_ebb, ebb, loc).and(Ok(ebb))
     }
 
+    fn add_alias(&mut self, src: Value, dest: Value, loc: &Location) -> Result<()> {
+        if self.aliases.insert(src, dest).is_some() {
+            err!(loc, "duplicate alias: {} -> {}", src, dest)
```

Comment:
Aliases are printed immediately before an instruction that uses them, so it is likely that there will be duplicates. We should only complain if there are *inconsistent* duplicates. That is, definitions of the same alias pointing to different values.